### ISSUE TITLE
[Bug Report][3.1.1] V-Select a11y error - role textbox should be listbox

### ISSUE THREAD

**Issue by landoephan** [vuetifyjs/vuetify#16538]
### Environment
**Vuetify Version:** 3.1.1
**Last working version:** 2.6.14
**Vue Version:** 3.2.40
**Browsers:** Chrome 109.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
- Open Codepen
- Inspect element `div` with class "v-field--appended"
- see role="textbox"

### Expected Behavior
There are no a11y violations.

### Actual Behavior
v-select violates a11y checks, e.g. "Elements must only use allowed ARIA attributes. Ensure ARIA attributes are allowed for an element's role"

### Reproduction Link
[https://codepen.io/landoephan/pen/ExpRNXZ](https://codepen.io/landoephan/pen/ExpRNXZ)

### Other comments
aria-role="textbox" is not allowed when using aria-expanded attribute. The role should be listbox or something similar. In vuetify 2 this worked with role="button". Also it would be necessary to give the option to add an "aria-labelledby" attribute to the div with class "v-field--appended".
